autonomy/deploy/generators/kubernetes/templates.py

```diff
@@ -306,10 +306,10 @@
 AGENT_SECRET_TEMPLATE: str = """
 apiVersion: v1
 stringData:
-    private_key: '{private_key}'
+  private_key: '{private_key}'
 kind: Secret
 metadata:
-  annotations:
+  annotations: {{}}
```

Comment:
This will print an empty mapping on the output  `annotations: {}`. Some yaml editors complained about this.